junipervpn/vpn.py

```diff
@@ -310,46 +311,45 @@ def stop(self):
 
 def main():
     parser = argparse.ArgumentParser(conflict_handler='resolve')
-    parser.add_argument('-h', '--host', type=str,
+    parser.add_argument('-h', '--host',
                         help='VPN host name')
-    parser.add_argument('-r', '--realm', type=str,
+    parser.add_argument('-r', '--realm',
                         help='VPN realm')
-    parser.add_argument('-u', '--username', type=str,
+    parser.add_argument('-u', '--username',
                         help='User name')
-    parser.add_argument('-p', '--pass_prefix', type=str,
```

Comment:
I'd be tempted to leave these as they are (i.e. with `type=str`) if that's OK - I guess I slightly prefer it because it means I don't have to remember what the default argument type is :)

Reply:
Thanks!